plugins/CoreAdminHome/API.php

```diff
@@ -335,4 +342,49 @@ private function getDatesToInvalidateFromString($dates, $period)
 
         return array($toInvalidate, $invalidDates);
     }
+
+    /**
+     * Show the JavaScript opt out code
+     *
+     * @param string $backgroundColor
+     * @param string $fontColor
+     * @param string $fontSize
+     * @param string $fontFamily
+     * @param bool $showIntro
+     * @param string $piwikUrl
```

Comment:
could you rename that parameter to `matomoUrl` everywhere, where it won't break BC?